timer.py

```diff
@@ -0,0 +1,8 @@
+import time
+
+
+def timer(func, parameter):
```

Comment:
Use type hints and return types -
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/